kratos/python_interface/application_importer.py

```diff
@@ -19,7 +20,7 @@ def ImportApplication(application, application_name, application_folder, caller)
         # print main_caller
         raise RuntimeError(msg)
     elif application_name not in Globals.RequestedApplications:  # This check is possibly redundant, as Python won't import the same module twice
-        print("Importing    " + application_name)
+        Logger.Print("Importing    " + application_name, label="")
```

Comment:
shouldn't this be `PrintInfo?`

Reply:
Thanks @philbucher . I just changed it and pushed.